vcpkg.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
+  "name": "bmq",
+  "version": "1.0.0",
```

Comment:
Should this version match our release version?

Reply:
Which version? 😛 I'll set it to the bmq SDK version for now, but we should probably have this version match a number that corresponds to some kind of source release.